bin/biomaj-cli.py

```diff
@@ -480,8 +480,7 @@ def main():
                 print("Bank name is missing")
                 sys.exit(1)
             bmaj = Bank(options.bank, options, no_log=True)
-            print('Log file: '+bmaj.config.log_file)
-            res = bmaj.remove_pending(options.release)
+            res = bmaj.remove_pending()
```

Comment:
Because in code, release passed as argument is not used.
Check the for loop in `remove_pending`